lang/python27/Portfile

```diff
@@ -264,6 +264,17 @@ platform darwin {
     }
 }
 
+platform darwin 8 {
+    if {$subport eq "${name}-bootstrap"} {
```

Comment:
Could you put this inside the subport block?

Reply:
sure